src/supervisor.patch/0002-Implement-SD_NOTIFY.patch

```diff
@@ -0,0 +1,207 @@
+From 3ab7d6f2007188ddce80fa3c1d6196cf888b3051 Mon Sep 17 00:00:00 2001
+From: Stepan Blyschak <stepanb@nvidia.com>
+Date: Mon, 29 Sep 2025 14:09:22 +0000
+Subject: [PATCH] Implement SD_NOTIFY
+
```

Comment:
Could you submit (just submit) a PR to [https://github.com/Supervisor/supervisor](https://github.com/Supervisor/supervisor) ?

Reply:
Submitted https://github.com/Supervisor/supervisor/pull/1699